.gitignore

```diff
@@ -0,0 +1,31 @@
+
+# Directories/files that may be generated by this project
+node_modules/
+dist
+build
+build-module
+build-style
+languages/*
+!languages/README.md
+coverage
+
+# Directories/files that may appear in your environment
+.DS_Store
+Thumbs.db
+wp-cli.local.yml
+*.sql
+*.tar.gz
+*.tgz
+*.zip
+.idea
+.vscode/
```

Comment:
💅 

In my opinion items like. `.DS_Store`, `Thumbs.db` `.idea`, `.vim` etc as well as `*.log`, `*.bak` and other files which are potentially affected by the developer environment or preference shouldn't be added to the projects' `.gitignore`.

Instead I'd suggest that each developer should be responsible of `gitignore global` as an example check here  https://gist.github.com/subfuzion/db7f57fff2fb6998a16c

Ideally, In the project it should be only the project based ignores. Otherwise each new developer using a different environment or naming for backup or log files would pollute with more and more stuff the project `gitignore` file.

Reply:
As I mentioned in OP, I'd like to improve `gitignore` template later. 
I added it here mostly, to show-case, what I plan to be there.

Initially, I was thinking of only adapting it so it works well for other repos that have different files structure, like `/assets`, etc.
But this is a really valid point. However, probably needs to be agreed with the Team, not to confuse people, but suddenly not ignore all the env files.

I created a separate issue to track that topic https://github.com/tomalec/generator-ventures/issues/3

Are you fine with leaving it as is, or should we rather remove `.gitignore` completely as YAGNI for today?

Reply:
Both options are fine 👌  Thanks!